src/test/java/tests/CartTests.java

```diff
@@ -7,11 +7,14 @@
 
     @Test
     public void testCartProductDelete() {
-        loginPage.openPage();//open page
-        loginPage.login(LOGIN_PAGE_USERNAME, LOGIN_PAGE_PASSWORD); //login
+        loginPage.openPage()//open page
```

Comment:
I think we can remove this comments like (//open page, //login and etc) because name of your methods are pretty good and it's easy to understand what they're doing inside.

Reply:
ok, done